frontend/src/constants/env.ts

```diff
@@ -3,4 +3,6 @@ export const ENVIRONMENT = {
 		process?.env?.FRONTEND_API_ENDPOINT ||
 		process?.env?.GITPOD_WORKSPACE_URL?.replace('://', '://8080-') ||
 		'',
+	isProduction: process?.env?.NODE_ENV === 'production',
```

Comment:
Why `isProduction` required are we using this anywhere.
I can see we already have `NODE_ENV`

Reply:
nopes not as of now... _(with Axolo)_

Reply:
Oh in that case let’s remove it, let;s add it when there’ll be the requirement _(with Axolo)_

Reply:
ok will remove this one _(with Axolo)_

Reply:
Pls update this

Reply:
1 min _(with Axolo)_

Reply:
done @pranshuchittora _(with Axolo)_